joomla/joomla-cms
# [fix] When installing Joomla! and mail sending fails you are not getting to the next screen.

Pull Request for Issue #11891 .
### Summary of Changes

The `libraries/legacy` tree isn't being included in the install app because its code was rewritten to not be dependent on it. Most likely this has been an issue since the PR that removed the "if JError exists use it for error handling else throw an Exception" logic from JMail. The error handling is trying to call a non-existing class.

Change installation/application/framework.php to require import.legacy.php instead of import.php.

Thanks @mbabker https://github.com/joomla/joomla-cms/issues/11891#issuecomment-244435068
### Testing Instructions
- Download this branch: https://github.com/zero-24/joomla-cms/archive/install_mail.zip
- upload it to the server
- add `$return = false;` here: https://github.com/joomla/joomla-cms/blob/staging/libraries/joomla/mail/mail.php#L133
- try to install joomla on the pre final step make sure you enable mail sending of the configuration.
##### Expected result

Installation succes should be independant of the setting send Email Configuration option.
##### Actual result

No matter which of the install-options you choose you will be getting this same Finalisation-screen until you set the send Email Configurations to No.
### Documentation Changes Required

None
